{
	"schema_version": "1.6.0",
	"id": "UBUNTU-CVE-2014-3589",
	"modified": "2025-01-13T10:21:06Z",
	"published": "2014-08-25T00:00:00Z",
	"related": [
		"CVE-2014-3589",
		"USN-3080-1",
		"USN-3090-1"
	],
	"details": "PIL/IcnsImagePlugin.py in Python Imaging Library (PIL) and Pillow before 2.3.2 and 2.5.x before 2.5.2 allows remote attackers to cause a denial of service via a crafted block size.",
	"affected": [
		{
			"package": {
				"ecosystem": "Ubuntu:14.04:LTS",
				"name": "pillow",
				"purl": "pkg:deb/ubuntu/pillow@2.3.0-1ubuntu3.2?arch=source&distro=trusty"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "2.3.0-1ubuntu3.2"
						}
					]
				}
			],
			"versions": [
				"2.2.1-1ubuntu2",
				"2.2.1-2ubuntu1",
				"2.2.1-3ubuntu2",
				"2.2.1-3ubuntu3",
				"2.2.1-3ubuntu4",
				"2.2.1-3ubuntu6",
				"2.3.0-1ubuntu1",
				"2.3.0-1ubuntu2",
				"2.3.0-1ubuntu3"
			],
			"ecosystem_specific": {
				"availability": "No subscription required",
				"binaries": [
					{
						"binary_name": "python-imaging",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-imaging-compat",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-imaging-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-imaging-doc",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-imaging-sane",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-imaging-sane-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-imaging-tk",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-imaging-tk-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-pil",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-pil-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-pil-dbgsym",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-pil-doc",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-pil.imagetk",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-pil.imagetk-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-pil.imagetk-dbgsym",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-sane",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-sane-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python-sane-dbgsym",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-imaging",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-imaging-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-imaging-sane",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-imaging-sane-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-imaging-tk",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-imaging-tk-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-pil",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-pil-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-pil-dbgsym",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-pil.imagetk",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-pil.imagetk-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-pil.imagetk-dbgsym",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-sane",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-sane-dbg",
						"binary_version": "2.3.0-1ubuntu3.2"
					},
					{
						"binary_name": "python3-sane-dbgsym",
						"binary_version": "2.3.0-1ubuntu3.2"
					}
				],
				"ubuntu_priority": "low"
			},
			"database_specific": {
				"source": "https://github.com/canonical/ubuntu-security-notices/blob/main/osv/cve/2014/UBUNTU-CVE-2014-3589.json"
			}
		}
	],
	"references": [
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/CVE-2014-3589"
		},
		{
			"type": "REPORT",
			"url": "https://ubuntu.com/security/notices/USN-3080-1"
		},
		{
			"type": "REPORT",
			"url": "https://www.cve.org/CVERecord?id=CVE-2014-3589"
		}
	]
}
